lib/accept_eula_test.py

```diff
@@ -30,199 +30,80 @@ def setUp(self):
+    @patch("util.exponential_backoff_cmd")
+    def test_accept_ert_eula_success(self, mock_exponential_backoff_cmd):
+        mock_exponential_backoff_cmd.side_effect = [login_success, accept_success]
         result = accept_eula.accept_ert_eula(self.settings)
-        self.assertEqual(result, ("Success", "", 0))
-
-    @patch("util.exponential_backoff")
-    def test_accept_ert_eula_fail(self, mock_exponential_backoff):
-        response = Mock(requests.Response)
-        response.status_code = 503
-        mock_exponential_backoff.return_value = (response, accept_eula.EULAResult.RETRY)
+        self.assertEqual(result, (
+            "Success; accepted: https://network.pivotal.io/api/v2/eulas/120 at: 2018-07-19", "", 0))
 
+    @patch("util.exponential_backoff_cmd")
+    def test_accept_ert_eula_fail(self, mock_exponential_backoff_cmd):
+        mock_exponential_backoff_cmd.side_effect = [login_success, accept_error]
         result = accept_eula.accept_ert_eula(self.settings)
-        self.assertEqual(result[2], 1)
-
-    @patch('requests.post')
-    def test_post_eula_success(self, mock_requests_post):
-        response = Mock(requests.Response)
-        response.status_code = 200
-        mock_requests_post.return_value = response
-
+        self.assertEqual(result, (
+            "Failed to accept ERT EULA; got message from Pivotal Network: foo error message", "", 1))
+
+    @patch("util.exponential_backoff_cmd")
+    def test_accept_stemcell_eula_success(self, mock_exponential_backoff_cmd):
+        mock_exponential_backoff_cmd.side_effect = [login_success, accept_success]
+        result = accept_eula.accept_stemcell_eula(self.settings)
+        self.assertEqual(result, (
+            "Success; accepted: https://network.pivotal.io/api/v2/eulas/120 at: 2018-07-19", "", 0))
+
+    @patch("util.exponential_backoff_cmd")
+    def test_accept_stemcell_eula_fail(self, mock_exponential_backoff_cmd):
+        mock_exponential_backoff_cmd.side_effect = [login_success, accept_error]
+        result = accept_eula.accept_stemcell_eula(self.settings)
+        self.assertEqual(result, (
+            "Failed to accept stemcell EULA; got message from Pivotal Network: foo error message", "", 1))
+
+    @patch("util.exponential_backoff_cmd")
+    def test_post_eula_success(self, mock_exponential_backoff_cmd):
+        mock_exponential_backoff_cmd.side_effect = [login_success, accept_success]
         result = accept_eula.post_eula(self.settings, 'my-awesome-product', 1337)
+        self.assertEqual(result, (
+            "accepted: https://network.pivotal.io/api/v2/eulas/120 at: 2018-07-19", "", 0))
 
-        self.assertEqual(result[0], response)
-        self.assertEqual(result[1], accept_eula.EULAResult.SUCCESS)
-
-        expected_headers = {
-            'Authorization': 'Token MY-TOKEN',
-            'Accept': 'application/json',
-            'Content-Type': 'application/json',
-            'User-Agent': 'PCF-Ecosystem-AWS-client'
```

Comment:
@cholick Thanks missed that one, do you have more context on the requirements around the `User-Agent` header?